yaml_files/cuba.yml

```diff
@@ -565,3 +555,59 @@ CUBA_KEYS:
     definition: Thermodynamic ensemble
     shape: [1]
     type: string
+
+  LATTICE_UC_ABC:
```

Comment:
LATTICE_VECTORS is more "text book" standard than LATTICE_UC_ABC, please revert to using the more standard terminology.